Sources/ArcGISToolkit/Components/Offline/OfflineMapInfo.swift

```diff
@@ -62,12 +62,13 @@ public extension OfflineMapInfo {
     var description: String { info.description }
     /// The URL of the portal item associated with the map.
     var portalItemURL: URL { info.portalItemURL }
+}
+
+extension OfflineMapInfo: Identifiable {
     /// The ID of the portal item associated with the map.
-    var portalItemID: Item.ID { .init(info.portalItemID)! }
+    public var id: Item.ID { .init(info.portalItemID)! }
```

Comment:
Not a big deal, but you could further clarify that this is the identifier of the portal item by using `PortalItem.ID` as the type.

```suggestion
    public var id: PortalItem.ID { .init(info.portalItemID)! }
```

Reply:
We have been using Item.ID throughout this code-base so I'd prefer to leave it for consistency.